Resources/AzureDevOps/StepTemplates/deploy-arm-template.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  AzureSubscription: 'SFA-CDH-Dev/Test (962cae10-2950-412a-93e3-d8ae92b17896)'  
+  SetSubscriptionScript: 'https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/PSScripts/Set-EsfaResourceGroupTags.ps1'
+  EnvironmentTag: 'DEV/Test'
+  ParentBusiness: ''
+  ServiceOffering: ''
+  ResourceGroup: ''
+  Location: 'West Europe'
+  ArmTemplateFolder: ''
+  ArmOutputVariableName: 'ARMOutputs'
+
+steps:
+- powershell:
```

Comment:
For consistency, can we decide if we are using a pipe at end of the powershell inline shortcut for a single line or not

Reply:
Decided on using the pipe char everywhere